spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/encryption/EncryptionController.java

```diff
@@ -136,7 +136,8 @@ public String decrypt(@PathVariable String name, @PathVariable String profiles,
 		}
 		catch (IllegalArgumentException | IllegalStateException e) {
 			if (logger.isErrorEnabled()) {
-				logger.error("Cannot decrypt key:" + name + ", value:" + data, e);
+				logger.error("Cannot decrypt key:" + name + ", value:" + data +
+					", Please verify if symmetric key is set correctly", e);
```

Comment:
What Symmetric Key means here is the `encrypt.key` property.

Reply:
I would suggest your replace `symmetric key` with `encrypt.key`